src/tasks/daily/daily_regional_runner.py

```
@@ -0,0 +1,53 @@
+            if enabled_outpost:
+                self._task.daily_routine.exchange_outpost_goods(
+                    target_areas=[area],
+                    keep_area_context=True,
+                )
+
+            materials_after_trade_buy = None
+            if enabled_buy and enabled_trade:
+                materials_after_trade_buy = lambda current_area: self._task.daily_buy.buy_staple_goods(
+                    target_areas=[current_area],
+                    keep_area_context=True,
+                )
```

Ruff (0.16.0)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `materials_after_trade_buy` as a `def`

(E731)